front-end/e-learning/src/components/words_choices/Delete.js

```diff
@@ -30,7 +28,7 @@ const Delete = ({
     setOpen(false);
     onSetState(0);
     onSetOpenNotification(true);
-    onSetMessage("Successfully removed word ID #" + category_word.id + "!");
+    onSetMessage("Successfully removed word " + category_word.word + "!");
```

Comment:
You can use backticks instead of adding a lot of "+" signs.

example:
```
onSetMessage("`Successfully removed word ${category_word.word}!`");
```

Reference:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals